### ISSUE TITLE
Add build guide

### ISSUE THREAD

**Issue by Programmerino** [DevonCrawford/LED-Music-Visualizer#1]
The code being open source is already great, but adding a concise build guide or just an image of the breadboard with a part list would be great for opening the possibility of others building your project.

**Comment by JOSHUALAZARO** [comment]
How come the code was deleted?

**Comment by Programmerino** [comment]
What code are you referring to?

**Comment by Conssole** [comment]
> The code being open source is already great, but adding a concise build guide or just an image of the breadboard with a part list would be great for opening the possibility of others building your project.

Agreed

**Comment by jj212th** [comment]
Would also love to see a complete build

**Comment by NicoleCollard** [comment]
I would also like a complete guide to building the signal processing.

**Comment by erozario1505** [comment]
Yes could we please get a picture of the bread board and a concise parts list with all the components used on the board.

**Comment by AnantaSrikar** [comment]
Please upload a brief schematics of the sensors used and the way they were connected.... it would help a lot of people who are trying to make it...... it's really awesome!

**Comment by DevShaded** [comment]
yes
